README.md

```diff
@@ -1 +1,23 @@
-# SeedAvatarStudio
\ No newline at end of file
+# SeedAvatarStudio
+
+
+
+## Setup
+
+The recommended Unity editor version is 2021.3.30f1c1, but other editor may likely work.
+
+
+## License
```

Comment:
This License section is typically used as the header of source code. Generally, we do not include it in README, since the separate LICENSE file has already made way more sense than a paragraph in README.